tests/run/i14693.scala

```diff
@@ -0,0 +1,9 @@
+object Test {
+  val a: Array[Long] = Array(1L)
+
+  def test(x: Any) = x match {
+    case Array(i: Long) => println("Success!")
+    case _              => println("Failure!") }
+
+  def main(args: Array[String]): Unit = test(a)
```

Comment:
One (or another) of vulpix's weak points is that it doesn't test output when the "check" file is absent. Nobody cares if you print failure.

Reply:
Right. Ir was a stray test, actually.